draft-irtf-t2trg-rest-iot.md

```diff
@@ -367,18 +372,22 @@ Therefore applications should not rely on their order; see Section 3.3 of {{?RFC
 ## Representations
 
 Clients can retrieve the resource state from an origin server or manipulate resource state on the origin server by transferring resource representations.
-Resource representations have a media type that tells how the representation should be interpreted by identifying the representation format used.
+Resource representations have a media-type that tells how the representation should be interpreted by identifying the representation format used.
```

Comment:
Is this a media-type or a content-type?

Reply:
I think both are OK/possible here, but I'd lean towards media-type for familiarity.

Reply:
content-type (media-type + optionally parameters)

maybe?

Reply:
SGTM. Fixed.